modules/300-prometheus/templates/aggregating-proxy/servicemonitor-mimir.yaml

```diff
@@ -0,0 +1,26 @@
+---
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  name: aggregating-proxy-mimir
+  namespace: d8-monitoring
+  {{- include "helm_lib_module_labels" (list . (dict "prometheus" "main")) | nindent 2 }}
+spec:
+  jobLabel: app
+  sampleLimit: 1500
+  endpoints:
+  - port: https
+    scheme: https
+    bearerTokenFile: /var/run/secrets/kubernetes.io/serviceaccount/token
+    tlsConfig:
+      insecureSkipVerify: true
+    honorLabels: true
+    relabelings:
+    - targetLabel: tier
+      replacement: cluster
+  selector:
+    matchLabels:
+      app: aggregating-proxy
+  namespaceSelector:
+    matchNames:
+    - d8-monitoring
```

Comment:
Probably, it is best to use a single service monitor with two endpoints then having two separated for the same set of pods. 

There is an example https://github.com/deckhouse/deckhouse/blob/main/modules/340-monitoring-kubernetes/templates/kubelet/service-monitor.yaml

Reply:
thank you for example, changed